types/prettier/index.d.ts

```diff
@@ -268,8 +268,8 @@ export interface SupportLanguage {
     group?: string;
     tmScope: string;
     aceMode: string;
```

Comment:
Some other keys can be optional, too.
See https://prettier.io/docs/en/api.html#prettiergetsupportinfo-version